eslint.config.mjs

```diff
@@ -0,0 +1,48 @@
+import globals from "globals";
+import js from "@eslint/js";
+import eslintConfigPrettier from "eslint-config-prettier";
+import eslintPluginPrettierRecommended from "eslint-plugin-prettier/recommended";
+import eslintPluginVue from 'eslint-plugin-vue'
+
+// disable import plugin for now (until https://github.com/import-js/eslint-plugin-import/issues/2948
+// and https://github.com/import-js/eslint-plugin-import/issues/2556 have been resolved)
+// import eslintPlugingImport from 'eslint-plugin-import';
+
+export default [
+  js.configs.recommended,
+  {
+    languageOptions: {
+      globals: {
+        ...globals.browser,
+      },
+      sourceType: "module",
+    },
+  },
+  //{
+  //  plugins: {
+  //    import: eslintPlugingImport,
+  //  },
+  //  files: ['**/*.js', '**/*.mjs', '**/*.ts'],
+  //  rules: {
+  //    ...eslintPlugingImport.configs.recommended.rules,
+  //    "import/order": [
+  //      "error",
+  //      {
+  //        "groups": ["type", "builtin", ["sibling", "parent"], "index", "object"],
+  //        "newlines-between": "never",
+  //        alphabetize: {order: "asc", caseInsensitive: true},
+  //      },
+  //    ],
+  //  },
+  //},
```

Comment:
remove?

Reply:
In this case I'd like to keep the disabled code because we probably want to enable it again in the future when the tickets also mentioned in comments are resolved.